esy-lib/Store.re

```diff
@@ -18,7 +18,7 @@ let maxStorePaddingLength = {
    * We reserve that amount of chars from padding so ocamlrun can be placed in
    * shebang lines
    */
-  let ocamlrunStorePath = "ocaml-n.00.000-########/bin/ocamlrun";
+  let ocamlrunStorePath = "ocaml-n.00.0000-########/bin/ocamlrun";
```

Comment:
@ManasJayanth Does the padding not need to be computed from the version number? That was my main reason for opening an issue rather than a PR...

Reply:
The padding needs to be constant regardless of the version.

Reply:
I don't understand. I thought that the purpose of the padding was to make sure that the _shebang_ was a constant width; If the OCaml version has a patch version _less than_ 1000, won't this break (just as it's currently broken for large patch numbers)?

Reply:
The part of the shebang path that needs to stay constant is the prefix - ie `~/.esy/3__..._/` The directory name that follows (ocaml-4.9.0 or 4.10.1000) can vary in length. Regarding the shebang length, it needs to stay the same length when relocated from one machine to another. 

If the package name is smaller, it's fine. The length would differ wrt another package name  of longer length (4.10.1000). What matters is that the binary artifacts reserve enough space when moved across machines.